docs/src/staking/stake-accounts.md

```diff
@@ -0,0 +1,97 @@
+a stake account using the command line tools](../cli/staking-operations.md#create-a-stake-account),
+a new keypair file is created first only to ensure that the stake account's
+address is new and unique.
+
+#### Understanding Account Authorities
+Each stake account has two signing authorities specified by their respective address,
+each of which is authorized to perform certain operations on the stake account.
+
+The *stake authority* is used to sign transactions for the following operations:
+ - Delegating stake
+ - Deactivating the stake delegation
+ - Splitting the stake account, creating a new stake account with a portion of the
+ funds in the first account
+ - Merging two undelegated stake accounts into one
+ - Setting a new stake authority
+
+The *withdraw authority* signs transactions for the following:
+ - Withdrawing un-delegated stake into a wallet address
+ - Setting a new withdraw authority
+ - Setting a new stake authority
+
+The stake authority and withdraw authority are set when the stake account is
+created, and they can be changed to authorize a new signing address at any time.
+The stake and withdraw authority can be the same address or two different
+addresses.
+
+The withdraw authority keypair holds more control over the account as it is
+needed to liquidate the tokens in the stake account, and can be used to reset
+the stake authority if the stake authority keypair becomes lost or compromised.
+
+Securing the withdraw authority against loss or theft is of utmost importance
+when managing a stake account.
+
+#### Multiple Delegations
+A stake account may only ever be delegated to one validator at a time. All of
+the tokens in the account are either delegated or un-delegated, or in the
+process of becoming delegated or un-delegated.  To delegate a fraction of your
+tokens to a validator, or to delegate to multiple validators, you must create
+multiple stake accounts.
+
+This can be accomplished by creating multiple stake accounts from a wallet
+address containing some tokens, or by creating a single large stake account
+and using the stake authority to split the account into multiple accounts
+with token balances of your choosing.
+
+The same stake and withdraw authorities can be assigned to multiple
+stake accounts.
+
+Two stake accounts that are not delegated and that have the same stake and
+withdraw authority can be merged into a single resulting stake account.
+
+#### Delegation Warmup and Cooldown
+When a stake account is delegated, or a delegation is deactivated, the operation
+does not take effect immediately.
+
+A delegation or deactivation takes several epochs to complete, with a fraction
+of the delegation becoming active or inactive at each epoch boundary after
+the transaction containing the instructions has been submitted to the cluster.
+
+There is also a limit on how much total stake can become delegated or
+deactivated in a single epoch, to prevent large sudden changes in stake across
+the network as a whole.
```

Comment:
We should probably mention that since warmup and cooldown are dependent on the behavior of other network participants, their exact duration is difficult to predict